@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/ihadss/fn_ihadssDraw.sqf

```diff
@@ -414,6 +414,9 @@ if (isNull fza_ah64_mycurrenttarget) then {
 if (currentweapon _heli in ["fza_m230", "fza_burstlimiter"] && !isNull laserTarget _heli) then {
     _targrange = format["*%1", round(_heli distance laserTarget _heli)];
 };
+if (currentWeapon _heli isKindOf ["fza_hellfire", configFile >> "CfgWeapons"] && !isNull laserTarget _heli) then {
+    _targrange = format["*%1", round(_heli distance (_heli getVariable "fza_ah64_currentlase"))];
```

Comment:
Not all hellfires are laser hellfires? Plus you can't get the range from just any laser iirc, it has to be the one from your LRFD hence the `laserTarget` above.

Reply:
I tried checking if ammo was laser type but it didn't want to work I figured when the lasers on you want a laser range you would turn off the laser if trying to shoot radar missiles
I can remove it if wanted

Reply:
Removing it is preferable, you can't get lasers ranges from just anything - it has to be the one you emit yourself. With other lases all you know is direction, not range